src/js/DarkTheme.js

```diff
@@ -9,6 +9,7 @@ const css_dark = [
 
 const DarkTheme = {
     configEnabled: undefined,
+    enabled: false,
```

Comment:
This would duplicate configEnabled

Reply:
Actually I'm not sure it does. It looked to me that configEnabled is not binary, so supports the option 'honour the system' so can't be relied upon for stating that dark theme was in fact enabled.

Reply:
Above statement would extract the value or get ConfigStorage `darkTheme` value.

Reply:
Which means configEnabled is not well named. It should be configSetting or similar. It can't be relied upon as to if it's actually applied or not... sure it'll have a value of two... but does that mean dark theme is active? I don't think it does because the test of isEnabled is not persisted...